silverback/application.py

```diff
@@ -75,7 +74,55 @@ def __init__(self, settings: Optional[Settings] = None):
 
     def on_startup(self) -> Callable:
         """
-        Code to execute on worker startup / restart after an error.
+        Code to execute on one worker upon startup / restart after an error.
+
+        Usage example::
+
+            @app.on_startup()
+            def do_something_on_startup(startup_state):
+                ...  # Reprocess missed events or blocks
+        """
+        return self.broker.task(task_name="silverback_startup")
```

Comment:
what are the pros/cons of moving the startup/shutdown handlers into separate tasks from the worker startup/shutdown events

Reply:
They cannot be combined.  They're very separate things. 

Having `on_startup` be a worker event means unexpected duplication of execution.  Having a separate task also allows us to feed the handler useful information, like `last_block_*`.  This is basically what we need for ApePay.

Renaming the previous handlers to be explicit worker event handlers implies their actual usage and is clearer, IMO.  They're two distinct things.

The only real con is we're changing up behavior here.  Anyone setting up worker state may need to refactor.

Reply:
yes.  `on_startup` will only run once.  `on_worker_startup` will run on every worker startup.